.ci/static-checks.sh

```diff
@@ -314,6 +314,23 @@ static_check_rust_arch_specific()
 	[ $? -eq 0 ] || die "crate not formatted by rustfmt."
 }
 
+# Install yamllint in the different Linux distributions
+install_yamllint()
+{
+	source /etc/os-release || source /usr/lib/os-release
+
+	package="yamllint"
+
+	case "$ID" in
+		centos|rhel) sudo yum -y install $package ;;
+		debian|ubuntu) sudo apt-get -y install $package ;;
+		fedora) sudo dnf -y install $package ;;
+		*) die "Please install yamllint on $ID" ;;
+	esac
+
+	[ -z "$have_yamllint_cmd" ] && info "Cannot install $package" && return
```

Comment:
I'm a touch confused - do you not have to re-evaluate `have_yamllint_cmd` (that is, do the `command -v` thing again), before re-testing this - otherwise it won't have changed afaict. Maybe I'm missing a subtlety.

Reply:
done :)